main.go

```diff
@@ -117,40 +116,15 @@ func main() {
 		os.Exit(1)
 	}
 
-	if err = (&controllers.OpenStackDataPlaneReconciler{
+	if err = (&controllers.OpenStackDataPlaneNodeSetReconciler{
 		Client:  mgr.GetClient(),
 		Scheme:  mgr.GetScheme(),
 		Kclient: kclient,
-		Log:     ctrl.Log.WithName("controllers").WithName("OpenStackDataPlane"),
+		Log:     ctrl.Log.WithName("controllers").WithName("OpenStackDataPlaneNodeSet"),
 	}).SetupWithManager(mgr); err != nil {
-		setupLog.Error(err, "unable to create controller", "controller", "OpenStackDataPlane")
+		setupLog.Error(err, "unable to create controller", "controller", "OpenStackDataPlaneNodeSet")
 		os.Exit(1)
 	}
-	if err = (&controllers.OpenStackDataPlaneRoleReconciler{
-		Client:  mgr.GetClient(),
-		Scheme:  mgr.GetScheme(),
-		Kclient: kclient,
-		Log:     ctrl.Log.WithName("controllers").WithName("OpenStackDataPlaneRole"),
-	}).SetupWithManager(mgr); err != nil {
-		setupLog.Error(err, "unable to create controller", "controller", "OpenStackDataPlaneRole")
-		os.Exit(1)
-	}
-	if err = (&controllers.OpenStackDataPlaneNodeReconciler{
-		Client:  mgr.GetClient(),
-		Scheme:  mgr.GetScheme(),
-		Kclient: kclient,
-		Log:     ctrl.Log.WithName("controllers").WithName("OpenStackDataPlaneNode"),
-	}).SetupWithManager(mgr); err != nil {
-		setupLog.Error(err, "unable to create controller", "controller", "OpenStackDataPlaneNode")
-		os.Exit(1)
-	}
-
-	if strings.ToLower(os.Getenv("ENABLE_WEBHOOKS")) != "false" {
-		if err = (&dataplanev1.OpenStackDataPlane{}).SetupWebhookWithManager(mgr); err != nil {
-			setupLog.Error(err, "unable to create webhook", "webhook", "OpenStackDataPlane")
-			os.Exit(1)
-		}
-	}
```

Comment:
I believe we need to enable webhooks for `OpenStackDataPlaneNodeSet`

Reply:
For what purpose?